Fix: several typos in the documentation (#929)
starknet-io/starknet-docs: components/Starknet/modules/cli/pages/starkli.adoc

interact with Starknet, by including the following options:

`--feeder_gateway_url`:: Sets the custom endpoint for read commands.

> `--gateway_url`:: Sets the custom endpoint for writing commands.

Reviewer: This change is incorrect. The sentence is talking about read/write commands.

* `ACCEPTED_ON_L2`
* `ACCEPTED_ON_L1`

> Refer to xref:architecture_and_concepts:Network_Architecture/transaction-life-cycle.adoc[this] section for more information about the transaction lifecycle.

Reviewer: ```suggestion
For more information, see xref:architecture_and_concepts:Network_Architecture/transaction-life-cycle.adoc[Transaction lifecycle].
```
